src/snowflake/snowpark/session.py

```diff
@@ -839,21 +841,21 @@ def add_requirements(
         paths in your requirements file in order to make Python scripts or directories containing Python scripts
         available to your UDF.
 
-        Pure Python packages that are not available in Snowflake will be pip installed locally and made available as an import
-        (via zip file on a remote stage). You can specify the remote stage as `persist_path` to create a persistent environment.
-        If you wish to use a specific version of pip, you can set the environment variable `PIP_PATH` to your pip
-        executable.
+        EXPERIMENTAL: Pure Python packages that are not available in Snowflake will be pip installed locally and made
+        available as an import (via zip file on a remote stage). You can specify the remote stage as `persist_path` to
+        create a persistent environment. If you wish to use a specific version of pip, you can set the environment
+        variable `PIP_PATH` to your pip executable. This feature is experimental, do not use it in production!
 
          Note that this function also supports addition of requirements via a `conda environment yaml file
          <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually>_`.
 
 
         Args:
             file_path: The path of a local requirement file.
-            force_push: Force upload Python packages with native dependencies.
-            force_install: Ignores environment present on persist_path and overwrites it with a fresh installation.
+            force_push: Force upload Python packages with native dependencies (experimental).
```

Comment:
I suggest add a `*` after `file_path` to make the 3 new arguments key-word arguments.

Reply:
Added